sokol: fix compiling gg and other graphical examples on OpenBSD
f53031feda33ab554cca0dc4df2fad06c0e3c332

Index: vlib/sokol/c/declaration.c.v
--- vlib/sokol/c/declaration.c.v.orig
+++ vlib/sokol/c/declaration.c.v
@@ -9,7 +9,7 @@ pub const used_import = 1
 
 #flag linux -lX11 -lGL -lXcursor -lXi -lpthread
 #flag freebsd -L/usr/local/lib -lX11 -lGL -lXcursor -lXi
-#flag openbsd -L/usr/X11R6/lib -lX11 -lGL -lXcursor -lXi
+#flag openbsd -I/usr/X11R6/include -L/usr/X11R6/lib -lX11 -lGL -lXcursor -lXi
 #flag windows -lgdi32
 
 $if windows {
